### DISCUSSION TITLE
Is `96.9.208.115:2092` supposed to be an upstream server?

### DISCUSSION THREAD

**Discussion by Jayman2000** [DescentDevelopers/Descent3#417] [Q&A]
[The wiki says:](https://github.com/DescentDevelopers/Descent3/wiki/Running-Descent-3#multiplayer-compatibility)

> You can connect to the following 1.5 servers:
>
> `96.9.208.115:2092` - Hosted by DescentMax7930, v1.5 Stable
> `64.44.131.41:2092` - Hosted by DescentMax7930, [Piccu Engine](https://github.com/InsanityBringer/PiccuEngine) (by Insan

I just tried connecting to `96.9.208.115:2092` using 4795388500777b78b5053e163d0869bf8644faae. Here’s what it looked like:

![According to the in-game server browser, the name of the server is “40PPS-Requires D3 Pi”.](https://github.com/DescentDevelopers/Descent3/assets/5579359/d26576ef-b9b6-4739-b39c-7f832d3ac7c6)

![When you connect to the server, it says “Colorado Piccu Engine server provided by DMax7930”.](https://github.com/DescentDevelopers/Descent3/assets/5579359/b01f9690-14f7-43f9-972d-a35fcccf0349)

Is `96.9.208.115:2092` still supposed to be running the upstream version of Descent 3? Or does the Wiki need to be updated to mention that `96.9.208.115:2092` is a [Piccu Engine](https://github.com/InsanityBringer/PiccuEngine/) server?

**Comment by JeodC** [comment]
Good question. D3 Devs don't maintain a server--have to ask DescentMax about that.

**Comment by Lgt2x** [comment]
If my cheap OVH VPS (EU-hosted though) can handle it I'll try hosting a server for development purposes that pulls D3 master every day. I'll keep you updated if I do that, it could help devs
